go.mod

```diff
@@ -114,14 +114,14 @@ require (
 	go.mongodb.org/mongo-driver v1.12.1
 	go.uber.org/multierr v1.11.0
 	go.uber.org/ratelimit v0.3.0
-	golang.org/x/crypto v0.19.0
-	golang.org/x/exp v0.0.0-20240119083558-1b970713d09a
-	golang.org/x/mod v0.14.0
-	golang.org/x/net v0.21.0
-	golang.org/x/oauth2 v0.16.0
-	google.golang.org/api v0.138.0
-	google.golang.org/grpc v1.59.0
-	google.golang.org/protobuf v1.32.0
+	golang.org/x/crypto v0.24.0
+	golang.org/x/exp v0.0.0-20231006140011-7918f672742d
+	golang.org/x/mod v0.17.0
+	golang.org/x/net v0.25.0
+	golang.org/x/oauth2 v0.18.0
+	google.golang.org/api v0.169.0
+	google.golang.org/grpc v1.64.0
```

Comment:
Your table shows that the patch was in 1.56.3. We were using 1.59.0 so no need to upgrade the release branch. This is a critical dependency for Dapr.

Reply:
I see. How about I bump it to match what runtime has for grpc for the release 1.13 branch. See here https://github.com/dapr/dapr/blob/release-1.13/go.mod
`[google.golang.org/grpc](http://google.golang.org/grpc) v1.60.1`

Reply:
thoughts @ItalyPaleAle ?

Reply:
Ok